inverse_velocity/loss_landscape.py

```diff
@@ -0,0 +1,106 @@
+import time
+import sys
+import os
+import numpy as np
+import json
+import pickle
+import torch.utils.checkpoint
+
+sys.path.append("/work2/08264/baagee/frontera/gns-main/")
```

Comment:
Don't use hardcoded paths especially on public repos. Others should be able to reproduce.